Sources/Client/TezosClient.swift

```diff
@@ -229,6 +229,62 @@ public class TezosClient {
                                         completion: completion)
   }
 
+  /**
+   * Retrieve ballots cast so far during a voting period.
+   */
+  public func getBallotsList(completion: @escaping ([[String: Any]]?, Error?) -> Void) {
+    let rpc = GetBallotsListRPC(completion: completion)
+    self.send(rpc: rpc)
+  }
+
+  /**
+   * Retrieve the expected quorum.
+   */
+  public func getExpectedQuorum(completion: @escaping (Int?, Error?) -> Void) {
+    let rpc = GetExpectedQuorumRPC(completion: completion)
+    self.send(rpc: rpc)
+  }
+
+  /**
+   * Retrieve the current period kind for voting.
+   */
+  public func getCurrentPeriodKind(completion: @escaping (PeriodKind?, Error?) -> Void) {
+    let rpc = GetCurrentPeriodKindRPC(completion: completion)
+    self.send(rpc: rpc)
+  }
+
+  /**
+   * Retrieve the sum of ballots cast so far during a voting period.
+   */
+  public func getBallots(completion: @escaping ([String: Any]?, Error?) -> Void) {
+    let rpc = GetBallotsRPC(completion: completion)
+    self.send(rpc: rpc)
+  }
+
+  /**
+   * Retrieve a list of proposals with number of supporters.
+   */
+  public func getProposalsList(completion: @escaping ([[String: Any]]?, Error?) -> Void) {
```

Comment:
same with the ballots above, using a concrete parsed type or a typealias rather than dictionaries would make this clearer